src/libcore/hash/sip.rs

```diff
@@ -22,7 +22,7 @@ use mem;
 /// This is currently the default hashing function used by standard library
 /// (eg. `collections::HashMap` uses it by default).
 ///
-/// See: https://131002.net/siphash/
+/// See: <https://131002.net/siphash>
```

Comment:
Nitpick: inconsistent trailing slash.

Reply:
both work

Reply:
The version without the trailing slash results in a 301 redirect.